### ISSUE TITLE
NVDA reports Unicode braille characters incorrectly

### ISSUE THREAD

**Issue by gjw32** [nvaccess/nvda#20632]
### Brief summary

If Unicode braille characters include dots456 and any combination of dots 7 and 8, then NVDA announces dots123 instead. For instance, ⢸⠀says "braille 1238" instead of "braille 4568". It shows up correctly on a braille display.

### Steps to reproduce

1. Type ⢸.
2. Read by character.
3. NVDA reads "braille 1238".

### Actual behavior

NVDA announces the character incorrectly.

### Expected behavior

NVDA should announce ⢸⠀as "braille 4568" and ⡸ as "braille 4567".

### NVDA logs, crash dumps and other attachments

No log necessary.

### NVDA type

installed copy

### NVDA version

NVDA 2026.1

### Windows version

Version	10.0.19045 Build 19045

### Does the issue still occur after restarting your computer?

I have restarted my computer and the issue still occurs

### If NVDA add-ons are disabled, is your problem still occurring?

I have restarted NVDA with add-ons disabled and the issue still occurs

### Does the issue still occur after you run the System Accessibility Repair Tool (COM Registration Fixing Tool in older versions) in NVDA's tools menu?

I have not run the System Accessibility Repair Tool

**Comment by seanbudd** [comment]
This can be fixed with the following diff.
@gjw32 - are you aware of any other symbols that are wrong? these were all added manually by hand, so some errors have been found over time. We have them all in [here](https://github.com/nvaccess/nvda/blob/master/source/locale/en/symbols.dic) if you have time to review them for correctness.

```diff
diff --git a/source/locale/en/symbols.dic b/source/locale/en/symbols.dic
index 000c8b76ca..6c287d471f 100644
--- a/source/locale/en/symbols.dic
+++ b/source/locale/en/symbols.dic
@@ -493,7 +493,7 @@ _	line	most
 ⡵	braille 1 3 5 6 7
 ⡶	braille 2 3 5 6 7
 ⡷	braille 1 2 3 5 6 7
-⡸	braille 1 2 3 7
+⡸	braille 4 5 6 7
 ⡹	braille 1 4 5 6 7
 ⡺	braille 2 4 5 6 7
 ⡻	braille 1 2 4 5 6 7
@@ -557,7 +557,7 @@ _	line	most
 ⢵	braille 1 3 5 6 8
 ⢶	braille 2 3 5 6 8
 ⢷	braille 1 2 3 5 6 8
-⢸	braille 1 2 3 8
+⢸	braille 4 5 6 8
 ⢹	braille 1 4 5 6 8
 ⢺	braille 2 4 5 6 8
 ⢻	braille 1 2 4 5 6 8
```

**Comment by gjw32** [comment]
The only other line I found that needs correcting is "⣸	braille 1 2 3 7 8".
